source/config/featureFlagEnums.py

```diff
@@ -139,6 +139,26 @@ def _displayStringLabels(self) -> dict["FontFormattingBrailleModeFlag", str]:
 		}
 
 
+class WordNavigationUnitFlag(DisplayStringEnum):
+	"""Enumeration for word navigation."""
+
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: Label for a method of word segmentation.
+			self.AUTO: _("Auto"),
+			# Translators: Label for a method of word segmentation.
+			self.UNISCRIBE: _("Uniscribe"),
```

Comment:
I fear "Uniscribe" is not understood by users. Could we find a more descriptive name for this option?

Reply:
I feel like this setting shouldn't even be exposed to the end user. 
We should just use the chinese method for chinese text, and uniscribe otherwise, completely under the hood.
Thoughts @michaelDCurran?